### PR TITLE
merging in webfundamentals content

### LINKED ISSUE #3544: content: Migrate Web Fundamentals Why Performance Matters to existing page Why Does Speed Matter
### What page(s) need to be updated?

https://web.dev/why-speed-matters/

### Why is this update needed?

Merge in useful content from https://developers.google.com/web/fundamentals/performance/why-performance-matters and redirect that page to Why Speed Matters.

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes #3544

Merging content from: https://developers.google.com/web/fundamentals/performance/why-performance-matters
To: https://web.dev/why-speed-matters/

The Web Fundamentals post had more links to research about the relationship between performance and business success, added those to this post.

### CODE REVIEW

File: src/site/content/en/fast/why-speed-matters/index.md [hunk 1]
```diff
@@ -3,28 +3,29 @@ layout: post
...
-
-{% Aside 'note' %}
-  Optimizing the speed of your site can also make it accessible to audiences who are on the slowest of network connections and may not have been able to access it or access it well previously.
-{% endAside %}
+Consumers increasingly rely on mobile to access digital content and services,
+and if you look at your site analytics,
+you'll probably see this story playing out in your own data.
+Consumers are also more demanding than they've ever been,
+and when they weigh up the experience on your site, they aren't just comparing you with your competitors,
```
Comment (jpmedley) [inline src/site/content/en/fast/why-speed-matters/index.md:23]:
```suggestion
and when they weigh the experience on your site, they aren't just comparing you with your competitors,
```

File: src/site/content/en/fast/why-speed-matters/index.md [hunk 2]
```diff
@@ -37,4 +38,96 @@ When it comes to user experience, speed matters. A [consumer study](https://www.
...
+Slow sites have a negative impact on revenue, and fast sites are shown to increase conversion rates.
+
+For [Mobify](http://resources.mobify.com/2016-Q2-mobile-insights-benchmark-report.html),
+every 100ms decrease in homepage load speed worked out to a 1.11% increase in session-based conversion,
+yielding an average annual revenue increase of nearly $380,000.
+Additionally, a 100ms decrease in checkout page load speed amounted to a 1.55% increase in session-based conversion,
+which in turn yielded an average annual revenue increase of nearly $530,000.
+
+When [AutoAnything reduced page load time by half](https://www.digitalcommerce360.com/2010/08/19/web-accelerator-revs-conversion-and-sales-autoanything/),
+they saw a boost of 12-13% in sales.
```
Comment (jpmedley) [inline src/site/content/en/fast/why-speed-matters/index.md:66]:
```suggestion
they saw a boost of 12% to 13% in sales.
```

File: src/site/content/en/fast/why-speed-matters/index.md [hunk 3]
```diff
@@ -37,4 +38,96 @@ When it comes to user experience, speed matters. A [consumer study](https://www.
...
+
+Retailer [Furniture Village](https://www.thinkwithgoogle.com/intl/en-gb/success-stories/uk-success-stories/furniture-village-and-greenlight-slash-page-load-times-boosting-user-experience/) audited their site speed and developed a plan to address the problems they found,
+leading to a 20% reduction in page load time and a 10% increase in conversion rate.
+
+## Performance is about user experience
+
+When it comes to user experience, speed matters.
+A [consumer study](https://www.ericsson.com/en/press-releases/2016/2/streaming-delays-mentally-taxing-for-smartphone-users-ericsson-mobility-report)
+shows that the stress response to delays in mobile speed are similar to that of watching a horror movie or solving a mathematical problem,
```
Comment (jpmedley) [inline src/site/content/en/fast/why-speed-matters/index.md:75]: LOL

File: src/site/content/en/fast/why-speed-matters/index.md [hunk 4]
```diff
@@ -37,4 +38,96 @@ When it comes to user experience, speed matters. A [consumer study](https://www.
...
+Public resources such as hospitals, clinics,
+and crisis centers have online resources that give users important and specific information that they need during a crisis.
+[While design is pivotal in presenting important information efficiently in stressful moments](https://aneventapart.com/news/post/eric-meyer-designing-for-crisis),
+the importance of delivering this information fast can't be understated.
+It's part of our job.
+
+## Test and improve your site
+
+The [Test My Site](https://www.thinkwithgoogle.com/feature/testmysite/)
+tool measure your site performance and gives recommendations for improvements.
```
Comment (jpmedley) [inline src/site/content/en/fast/why-speed-matters/index.md:131]:
```suggestion
tool measures your site performance and gives recommendations for improvements.
```

File: src/site/content/en/fast/why-speed-matters/index.md [hunk 5]
```diff
@@ -3,28 +3,29 @@ layout: post
...
-
-{% Aside 'note' %}
-  Optimizing the speed of your site can also make it accessible to audiences who are on the slowest of network connections and may not have been able to access it or access it well previously.
-{% endAside %}
+Consumers increasingly rely on mobile to access digital content and services,
+and if you look at your site analytics,
+you'll probably see this story playing out in your own data.
+Consumers are also more demanding than they've ever been,
+and when they weigh up the experience on your site, they aren't just comparing you with your competitors,
+they're rating you against the best in class services they use every day.
```
Comment (kaycebasques) [inline src/site/content/en/fast/why-speed-matters/index.md:24]:
```suggestion
they're rating you against the best-in-class services they use every day.
```

File: src/site/content/en/fast/why-speed-matters/index.md [hunk 6]
```diff
@@ -37,4 +38,96 @@ When it comes to user experience, speed matters. A [consumer study](https://www.
   </blockquote>
 </figure>
 
-Many brands are already investing in mobile speed and achieving great results. In just [one example](https://www.thinkwithgoogle.com/intl/en-gb/success-stories/uk-success-stories/furniture-village-and-greenlight-slash-page-load-times-boosting-user-experience/), retailer Furniture Village audited their site speed and developed a plan to address the problems they found, leading to a 20% reduction in page load time and a 10% increase in conversion rate.
+Performance plays a major role in the success of any online venture
+and high-performing sites engage and retain users better than low-performing ones.
```
Comment (kaycebasques) [inline src/site/content/en/fast/why-speed-matters/index.md:42]:
```suggestion
Performance plays a major role in the success of any online venture.
High-performing sites engage and retain users better than low-performing ones.
```

File: src/site/content/en/fast/why-speed-matters/index.md [hunk 7]
```diff
@@ -37,4 +38,96 @@ When it comes to user experience, speed matters. A [consumer study](https://www.
...
+
+## Performance is about people
+
+Poorly performing sites and applications can also pose real costs for the
+people who use them.
+
+[As mobile users continue to make up a larger portion of internet users
+worldwide](http://gs.statcounter.com/platform-market-share/desktop-mobile-tablet),
+it's important to bear in mind that many of these users access the web through
+mobile LTE, 4G, 3G and even 2G networks.
```
Comment (kaycebasques) [inline src/site/content/en/fast/why-speed-matters/index.md:107]:
```suggestion
mobile LTE, 4G, 3G, and even 2G networks.
```

File: src/site/content/en/fast/why-speed-matters/index.md (hunk 4 above)
Comment (kaycebasques) [reply src/site/content/en/fast/why-speed-matters/index.md:131]: I'm going to do a quick rewrite of this section to focus on modern tools

File: src/site/content/en/fast/why-speed-matters/index.md [hunk 8]
```diff
@@ -37,4 +38,96 @@ When it comes to user experience, speed matters. A [consumer study](https://www.
...
+[While design is pivotal in presenting important information efficiently in stressful moments](https://aneventapart.com/news/post/eric-meyer-designing-for-crisis),
+the importance of delivering this information fast can't be understated.
+It's part of our job.
+
+## Test and improve your site
+
+The [Test My Site](https://www.thinkwithgoogle.com/feature/testmysite/)
+tool measure your site performance and gives recommendations for improvements.
+
+The [Fast load times](/fast/) section of this site gives technical tips to help you achieve your best performance.
```
Comment (kaycebasques) [inline src/site/content/en/fast/why-speed-matters/index.md:133]:
```suggestion
## Get started with improving your website speed {: #get-started }

Read up on the [Core Web Vitals](/vitals/#core-web-vitals) to learn about the metrics
that Google believes all websites should focus on.

<blockquote>
  <p>
	  Today, we’re building on this work and providing an early
	  look at an upcoming Search ranking change that incorporates
	  these page experience metrics. We will introduce a new signal
	  that combines Core Web Vitals with our existing signals for page
	  experience to provide a holistic picture of the quality of a user’s
	  experience on a web page.
  </p>
  <cite>
    <a href="https://webmasters.googleblog.com/2020/05/evaluating-page-experience.html">
      Evaluating page experience for a better web
    </a>, Official Google Webmaster Central Blog
  </cite>
</blockquote>

Then check out [Fast load times](/fast/) for lots of tips and tricks
related to getting fast and staying fast.
```

**Comment by kaycebasques** [comment]
And thank you for migrating the content @rachelandrew, the page is much more compelling now

**Comment by rachelandrew** [comment]
PR for redirection: https://github.com/google/WebFundamentals/pull/8900

**Pull request merged**